src/main/resources/application.yaml

```diff
@@ -18,6 +18,18 @@ spring:
     password: ${DB_PASSWORD}
     properties:
       charSet: UTF-8
+  mail:
+    host: ${SMTP_HOST}
+    username: ${SMTP_USERNAME}
+    password: ${SMTP_PASSWORD}
```

Comment:
are those env vars set?

Reply:
if not, `test-connection` flag should be false to allow service to start up and env vars set up in the following PR